### PR TITLE
Ft feedback week 2 day 2

### PR DESCRIPTION
**Pull request by shalu93**
#### What does this PR do?
Added a test DB, Modified tests and scripts

#### Description of Task to be completed?
- added test db
- Modified test scripts
- Handled breaking condition for signup.
- Error messages changed.
- added 5 API documentation on swagger.yml
- changed versions for challenge 2 and 3 as v1 and v2.

#### How should this be manually tested?
this can be manually tested via postman and by running npm run test

#### Any background context you want to provide?
no

#### What are the relevant pivotal tracker stories?
#165560357 , #165592388

#### Screenshots (if appropriate)
no

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,16 +45,18 @@ The app is hosted on heroku link for the same is : https://svas123.herokuapp.com
 | /POST | /api/v1/auth/signup | Creating a User |
 | /POST | /api/v1/auth/signin | Admin/Staff/Client Login |
 | /GET | /api/v1/accounts | Get list of all accounts |
-| /POST | /api/v1/accounts | Create a net account |
+| /POST | /api/v1/accounts | Create a neW account |
```
Comment (brandeddavid) [inline README.md:48]: Make W a small letter

File: server/controllers/accountController.js [hunk 2]
```diff
@@ -68,14 +68,19 @@ export default class authUsers{
                 status: 400,
                 message: 'only numbers are allowed in the account number field'
             });
-        }
+        } else {
         var AcctId = parseInt(req.params.accountNumber);
```
Comment (brandeddavid) [inline server/controllers/accountController.js:72]: Use a more descriptive variable name here.

File: server/controllers/accountController.js [hunk 3]
```diff
@@ -193,26 +199,25 @@ export default class authUsers{
                     status: 400,
                     message: 'only numbers are allowed in the account number field'
                 });
-            }
+            } else {
 
             if(req.body.status.toLowerCase() != 'active') {
```
Comment (brandeddavid) [inline server/controllers/accountController.js:204]: I think you could reduce lines 204 to 206 to a single line using logical operators.

Comment (brandeddavid) [reply server/controllers/accountController.js:204]: Also look into other parts of your code you could reduce the if statements by using logical operators.

File: server/routes/acctRoutes.js [hunk 4]
```diff
@@ -4,12 +4,12 @@ import auth from '../authorization/auth';
 
 const AcctRouter=Router();
```
Comment (brandeddavid) [inline server/routes/acctRoutes.js:5]: Make this variable name more descriptive

File: server/routes/acctRoutes.js [hunk 5]
```diff
@@ -4,12 +4,12 @@ import auth from '../authorization/auth';
...
 
-AcctRouter.get('/accounts',auth.authorization,accounts.getAcctInfo);
-AcctRouter.post('/accounts',auth.authorization,accounts.createAccount);
-AcctRouter.patch('/account/:accountNumber',auth.authorization,accounts.updateAccount);
-AcctRouter.delete('/accounts/:accountNumber',auth.authorization,accounts.deleteAccount);
+AcctRouter.get('/v1/accounts',auth.authorization,accounts.getAcctInfo);
```
Comment (brandeddavid) [inline server/routes/acctRoutes.js:7]: Are all these v1s part of challenge 2 work?

**Review by brandeddavid — CHANGES_REQUESTED** [review]

File: server/routes/acctRoutes.js (hunk 5 above)
Comment (shalu93) [reply server/routes/acctRoutes.js:7]: yes

File: README.md (hunk 1 above)
Comment (shalu93) [reply README.md:48]: changed

File: server/controllers/accountController.js (hunk 2 above)
Comment (shalu93) [reply server/controllers/accountController.js:72]: changed to AccountNumber

File: server/controllers/accountController.js (hunk 3 above)
Comment (shalu93) [reply server/controllers/accountController.js:204]: req.body.type.toLowerCase() != 'saving' && req.body.type.toLowerCase() != 'current'

changed at all the places that i found

File: server/routes/acctRoutes.js (hunk 4 above)
Comment (shalu93) [reply server/routes/acctRoutes.js:5]: changed it to AccountRouter

**Pull request merged**